pkg/fpgacontroller/patcher/patcher.go

```diff
@@ -21,10 +22,10 @@ import (
 	"sync"
 	"text/template"
 
+	"github.com/go-logr/logr"
```

Comment:
why `klog` is not usable?

Reply:
For some reason K8s's controller-runtime seems to use logr internally instead of klog:
https://github.com/kubernetes-sigs/controller-runtime/blob/5ef5ed9bcde997a4388b2697b5ff0460bb0ee0d0/alias.go#L123-L134

So, I decided to align with it.

Reply:
I think it'd be better to align all our content with klog. AFAICS it implements `logr.Logger` too so we should be able to replace zap with it

Reply:
Ok, replaced `zap` with the `klogr` implementation of `logr.Logger`.